HW2.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+//1
+let text = "";
+for(let i = 1; i<=100;i++){
+
+    if (i%3 === 0 && i%5 === 0 ){
+        text+= " LOLKEK"
+    }
+       else if (i%3 === 0){
+        text+= " KEK"
+    }        
+        else if (i%5 === 0){
+            text+= " LOL"
+
+        }
+        else{ 
+            text+= ` ${i}`}
+}
+console.log(text);
+let text1 = "";
+let text2 = "";
+//2
+const value = 10;
+let i = 0;
+if (Number.isNaN(value)){
```

Comment:
Зараз цей код не працює, якщо його запустити, виведетеся console.log("Таке чуство шо Бог десь наказує нас за шось"), хочу value у нас число, мабуть планувалось:
```suggestion
if (!Number.isNaN(value)){
```
?